helm/templates/webapp.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nexus-webapp
+spec:
+  type: ClusterIP
+  ports:
+  - port: 8083
+    name: webapp
+  selector:
+    app: nexus-webapp
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nexus-webapp
+  name: nexus-webapp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nexus-webapp
+  template:
+    metadata:
+      labels:
+        app: nexus-webapp
+    spec:
+      containers:
+      - image: {{ .Values.webapp.singleNode.image }}
+        imagePullPolicy: Always
+        name: nexus-webapp
+        env:
+          - name: CASSANDRA_CONTACT_POINTS
```

Comment:
I don't think any of these environment variables are actually consumed anymore since the `docker-entrypoint.sh` script was removed (with maybe the exception of `MASTER`, `SPARK_LOCAL_IP`, and `PYSPARK_SUBMIT_ARGS`?)